exonum/src/events/noise/tests.rs

```diff
@@ -161,15 +162,11 @@ fn test_noise_handshake_errors_ee_empty() {
     ));
     let (_, listener_err) = wait_for_handshake_result(addr, &params, bogus_message, None);
 
-    assert!(
-        listener_err
-            .unwrap_err()
-            .description()
-            .contains("WrongMessageLength")
-    );
+    listener_err.unwrap()
```

Comment:
Doesn't failure provides the description?

Reply:
nope, and is there some reason why we need use it instead of should panic?

Reply:
Now it's not necessary, but early versions of this tests checked two errors instead of one.